ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/util/EpochProcessorUtil.java

```diff
@@ -72,7 +72,7 @@ public static void updateCrosslinks(BeaconState state) throws BlockValidationExc
               .getLatest_crosslinks()
               .set(
                   shard.intValue(),
```

Comment:
I realize this wasn't necessarily part of your change, but it made it into the diff. 🤷‍♀️Same concern as all the previous intValues.